ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -465,6 +465,35 @@ def test_tensorflow_negative(
     )
 
 
+# lgamma
+@handle_frontend_test(
+    fn_tree="tensorflow.math.lgamma",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid"),
```
`valid` should be able to return all the supported dtypes for the function.